portable/IAR/AtmelSAM7S64/lib_AT91SAM7X256.h

```diff
@@ -1,1469 +1,1396 @@
-   ***************************************************************************** */
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_DBGU_InterruptEnable
-//* \brief Enable DBGU Interrupt
-//*----------------------------------------------------------------------------
-__inline void AT91F_DBGU_InterruptEnable(
-        AT91PS_DBGU pDbgu,   // \arg  pointer to a DBGU controller
-        unsigned int flag) // \arg  dbgu interrupt to be enabled
-{
+*               SOFTWARE API FOR DBGU
+***************************************************************************** */
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_DBGU_InterruptEnable */
+/** \brief Enable DBGU Interrupt */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_DBGU_InterruptEnable( AT91PS_DBGU pDbgu,  /* \arg  pointer to a DBGU controller */
+                                              unsigned int flag ) /* \arg  dbgu interrupt to be enabled */
+    {
         pDbgu->DBGU_IER = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_DBGU_InterruptDisable
-//* \brief Disable DBGU Interrupt
-//*----------------------------------------------------------------------------
-__inline void AT91F_DBGU_InterruptDisable(
-        AT91PS_DBGU pDbgu,   // \arg  pointer to a DBGU controller
-        unsigned int flag) // \arg  dbgu interrupt to be disabled
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_DBGU_InterruptDisable */
+/** \brief Disable DBGU Interrupt */
+/**---------------------------------------------------------------------------- */
+    __inline void AT91F_DBGU_InterruptDisable( AT91PS_DBGU pDbgu,  /* \arg  pointer to a DBGU controller */
+                                               unsigned int flag ) /* \arg  dbgu interrupt to be disabled */
+    {
         pDbgu->DBGU_IDR = flag;
-}
-
-//*----------------------------------------------------------------------------
-//* \fn    AT91F_DBGU_GetInterruptMaskStatus
-//* \brief Return DBGU Interrupt Mask Status
-//*----------------------------------------------------------------------------
-__inline unsigned int AT91F_DBGU_GetInterruptMaskStatus( // \return DBGU Interrupt Mask Status
-        AT91PS_DBGU pDbgu) // \arg  pointer to a DBGU controller
-{
+    }
+
+/**---------------------------------------------------------------------------- */
+/** \fn    AT91F_DBGU_GetInterruptMaskStatus */
+/** \brief Return DBGU Interrupt Mask Status */
+/**---------------------------------------------------------------------------- */
+    __inline unsigned int AT91F_DBGU_GetInterruptMaskStatus( /* \return DBGU Interrupt Mask Status */
+        AT91PS_DBGU pDbgu )                                  /* \arg  pointer to a DBGU controller */
```

Comment:
Should this be one line?

Reply:
Answered [here](https://github.com/FreeRTOS/FreeRTOS-Kernel/pull/768#discussion_r1313523527)